helm/stacks-blockchain/README.md

```diff
@@ -0,0 +1,136 @@
+# Stacks Blockchain
+
+[stacks-blockchain](https://github.com/blockstack/stacks-blockchain) is an layer-1 blockchain that connects to Bitcoin for security and enables decentralized apps and predictable smart contracts.
+
+## TL;DR
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain
+```
+
+## Introduction
+
+This chart bootstraps a [stacks-blockchain](https://github.com/blockstack/stacks-blockchain) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.12+
+- Helm 2.12+ or Helm 3.0-beta3+
+
+## Installing the Chart
+
+To install the chart with the release name `my-release` and run the node as a **follower**:
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain
+```
+
+To install the chart with the release name `my-release` and run the node as a **miner** using your private key [generated from the instructions on this page](https://docs.blockstack.org/mining):
+
+```bash
+$ helm repo add blockstack https://charts.blockstack.xyz
+$ helm install my-release blockstack/stacks-blockchain --set config.node.miner=true --set config.node.seed="REPLACE-WITH-YOUR-PRIVATE-KEY"
+```
+
+These commands deploy the Stacks Blockchain node on a Kubernetes cluster in the default configuration.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following tables lists the configurable parameters of the stacks-blockchain chart and their default values.
+
+| Parameter | Description | Default |
+|-|-|-|
+| nameOverride | String to partially override stacks-blockchain.fullname template (will maintain the release name) | nil |
+| fullnameOverride | String to fully override stacks-blockchain.fullname template | nil |
+| node.image.repository | stacks-blockchain container image repository | blockstack/stacks-blockchain |
+| node.image.pullPolicy | stacks-blockchain container image pull policy | IfNotPresent |
+| node.image.tag | stacks-blockchain container image tag | v23.0.0.10-krypton |
```

Comment:
For public release, i propose we use the `latest` tag, else we'll have to keep updating this for every new release until mainnet. post-mainnet, we can revert to a stable versioned tag.